packages/javascript/bh-shared-ui/src/components/AssetGroupFilters/AssetGroupFilters.tsx

```diff
@@ -80,28 +57,38 @@ const AssetGroupFilters: FC<Props> = ({ filterParams, handleFilterChange, member
     };
 
     const active = !!filterParams.primary_kind || !!filterParams.custom_member;
-    const activeStyles = active ? classes.activeFilters : '';
 
     return (
         <Box
             p={1}
-            className={activeStyles}
-            bgcolor={theme.palette.neutral.secondary}
             component={Paper}
+            bgcolor={theme.palette.neutral.secondary}
             elevation={0}
             marginBottom={1}
             data-testid='asset-group-filters-container'>
             <Button
                 onClick={() => setDisplayFilters((prev) => !prev)}
                 data-testid='display-filters-button'
-                style={{ width: '100%', marginBottom: '12px' }}>
+                className={clsx('w-full', active && 'font-bold')}>
```

Comment:
Clever using the tailwind class thats getting shipped with doodle! Now we just need to introduce it to bhe/bhce!

Reply:
Yea maybe we can try using the tailwind classes wherever possible going forward now that the stylesheet is being included in the document.